docs/content/concepts/executor.en.md

```diff
@@ -4,19 +4,65 @@ draft: false
 weight: 23
 ---
 
-When you create a function, you can specify an executor for a function. An executor controls how function pods are created and what capabilities are available for that executor type.
+A key aspect of performance latency in any high-throughput Serverless application is activation latency for incoming functions, since every single function effectively may require instantiation of a new container in a naive implementation.
 
-## Pool-based executor
+That said, not all functions require low-latency: some functions may have a very long computationally intensive cold-start, which is amortized over an even longer running batch job time.
```

Comment:
This seems to be conflating 3 things:
1. functions that don't need fast cold starts
2. functions that can't cold-start quickly
3. batch jobs.  I'm not sure how this part relates to fission.